searchlib/src/vespa/searchlib/tensor/hnsw_index.cpp

```diff
@@ -287,8 +325,8 @@ HnswIndex::add_document(uint32_t docid)
     while (search_level >= 0) {
         // TODO: Rename to search_level?
         search_layer(input, _cfg.neighbors_to_explore_at_construction(), best_neighbors, search_level);
-        auto neighbors = select_neighbors(best_neighbors.peek(), max_links_for_level(search_level));
-        connect_new_node(docid, neighbors, search_level);
+        auto neighbors = select_neighbors(best_neighbors.peek(), _cfg.max_links_at_hierarchic_levels());
+        connect_new_node(docid, neighbors.used, search_level);
```

Comment:
I guess we can remove the TODO on the next line?

Reply:
right!